README.md

```diff
@@ -88,6 +88,30 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ### Linux Debian package (.deb)
 
+`apt-get` support is available for Ubuntu Bionic Beaver (18.04) and Hirsute 
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+#### Bionic
+
+```shell
+curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod
+sudo apt-get update
+sudo apt-get install microsoft-git
```

Comment:
This package name should be different for GCM core.

Reply:
Oh dear.